components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/CacheBackedIdPMgtDAO.java

```diff
@@ -517,6 +517,58 @@ public IdentityProvider getIdPByRealmId(String realmId, int tenantId,
         return identityProvider;
     }
 
+    /**
+     * Get the enabled IDP of the given realm id.
+     *
+     * @param realmId Realm ID of the required identity provider.
+     * @param tenantId Tenant ID of the required identity provider.
+     * @param tenantDomain Tenant domain of the required identity provider.
```

Comment:
Check formatting here. Better to have param descriptions inline

Reply:
Check similar places

Reply:
Fixed with [056e029](https://github.com/wso2/carbon-identity-framework/pull/4991/commits/056e02961a66d2b8b8e6988d7af2f8a054a6dddb)